src/main/webapp/Portal.html

```diff
@@ -114,26 +128,19 @@
 			head.load(cdnEndpoint + "js/xss.min.js");
 			head.load(cdnEndpoint + "generated/pica.min.js");
 			head.load("//www.google-analytics.com/analytics.js");
-			if (processEnvironment === "production") {
-				head.load(cdnEndpoint + "generated/react.production.min.js");
-				head.load(cdnEndpoint + "generated/react-dom.production.min.js");
-				head.load(cdnEndpoint + "generated/react-query.production.min.js");
-			} else {
-				head.load(cdnEndpoint + "generated/react.development.js");
-				head.load(cdnEndpoint + "generated/react-dom.development.js");
-				head.load(cdnEndpoint + "generated/react-query.development.js");
-			}
+			head.load(reactPath);
+			head.load(reactDomPath);
+			head.load(reactQueryPath);
 			head.load(cdnEndpoint + "generated/react-transition-group.min.js");
 			head.load(cdnEndpoint + "generated/prop-types.min.js");
 			head.load(cdnEndpoint + "generated/react-measure.js");
-			head.load(cdnEndpoint + "generated/react-tooltip.min.js");
```

Comment:
In SRC we're using react tooltip 4 and react tooltip 4 doesn't seem to distribute a UMD build, so we're bundling it with SRC.